tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
```

Comment:
I can remove this for now. You are right that if some other test fails so that it leaves the port in use there is some other bug elsewere.